tcResources/grc/bibles/ugnt/v0.5/index.json

```diff
@@ -561,7 +561,7 @@
     "16": {
       "1": 15,
       "2": 5,
-      "3": 23,
+      "3": 0,
```

Comment:
So we have an empty verse for 16:3?

Reply:
This is a footnoted verse (all content is in footnote) - new word count should be 0.

Reply:
I mean alignment count